tlsr9/common/b9x_rf_power.c

```diff
@@ -116,46 +116,57 @@ const uint8_t b9x_tx_pwr_lt[] = {
 #elif CONFIG_SOC_RISCV_TELINK_TL321X
 /* TX power TL321X lookup table */
 const uint8_t tlx_tx_pwr_lt[] = {
```

Comment:
The size of array tlx_tx_pwr_lt is related to TX_POWER_MIN(MAX). 
`#define TLX_TX_POWER_MIN                    (-30)`
`#define TLX_TX_POWER_MAX                    (9)`
For the B9x series, it contains 40 values. If you need to increase it for the TLx series, you also need to modify b9x_rf_power.h accordingly; otherwise, some elements in the array will be invalid.

Reply:
Adjust to 51 valued.